### ISSUE TITLE
live script open in matlab button

### ISSUE THREAD

**Issue by claire-braboszcz** [sccn/sccn.github.io#37]
@arnodelorme could you send me the link where you found the piece of script to do that?
I can't find it again. Thanks!

**Comment by arnodelorme** [comment]
We have to discuss with Dung.

For now, we can put it in code format for people to copy and paste

```matlab
openExample('nnet/VisualizeActivationsOfAConvolutionalNeuralNetworkExample’)
```

**Comment by arnodelorme** [comment]
open(fullfile(fileparts(which('eeglab')), 'tutorial_scripts', 'LiveScripts', 'dipfit_fit_dipole_on_scalp_map.mlx'))

**Comment by arnodelorme** [comment]
We will have to fix the path

**Comment by arnodelorme** [comment]
https://www.mathworks.com/help/deeplearning/ug/transfer-learning-with-deep-network-designer.html

**Comment by arnodelorme** [comment]
This is an example of a command to put in the pop-up window

eeglabp = fileparts(which('eeglab.m')); open(fullfile(eeglabp, 'tutorial_scripts', 'LiveScripts', 'dipfit_source_reconstruction.mlx'));

**Comment by dungscout96** [comment]
I put up an example in https://eeglab.org/test.html. (clear cache before you do since it requires reloading of the css). You can go to the page content the see the two line you can copy and paste where you want to have it, changing the command accordingly.

**Comment by arnodelorme** [comment]
This is great. Can we add a comment above such as “Please copy and paste this command in your Matlab command window”.

**Comment by dungscout96** [comment]
Added. Also add more styling and updated it so that you can have multiple commands in one page. And made the syntax simpler. Use this and replace <the matlab command> with the command:

<a style="cursor: pointer" onclick="showModal(this)" data-command="the matlab command">Show MATLAB command</a>

**Comment by arnodelorme** [comment]
This is great. Is it possible to make a button instead of a link?

**Comment by dungscout96** [comment]
Yep. Just need to change the <a> tag to <button> tag. I updated the test file to show an example. I'll work on some styling to make the button looks a bit nicer:)

**Comment by arnodelorme** [comment]
Very nice already.

**Comment by dungscout96** [comment]
Bug:
* Text overflow when screen is small
* Right alignment

**Comment by dungscout96** [comment]
Fixed text overflow issue in fb37e252c701535aafc2f50aa3807a38bdaf730f. For right alignment use: `class="float-right"` in the `<button>` component.

**Comment by arnodelorme** [comment]
Cool. Thanks Dung,
